docker/develop/docker-compose.yml

```diff
@@ -36,7 +36,7 @@ services:
       - redis_data:/data
 
   postgresql:
-    image: postgres:10.3
+    image: postgres:10-bullseye
```

Comment:
Is there any reason not to use 15?

Reply:
I initially didn't want to do a major upgrade (see initial PR comment), but after discussing in IRC we decided, that it doesn't matter for a development environment, as the database probably gets wiped regularly anyway and there shouldn't be any important data.

So you need to delete the volume and let it recreate, the easiest way is probably running `script/diaspora-dev docker-compose down -v` followed by `script/diaspora-dev setup` (there are other ways which selectively only delete the postgres volume, or you can try to migrate/upgrade the data, but I leave that to the devs if they want to do that).